document/web-api/index.bs

```diff
@@ -158,6 +162,8 @@ The [=deserialization steps=], given |serialized| and |value|, are:
     1. Set |value|.\[[Bytes]] to |bytes|.
 
     1. If |targetRealm|'s corresponding [=agent cluster=] is not |serialized|.\[[AgentCluster]], then throw a "<a exception>DataCloneError</a>" {{DOMException}}.
+    
+    1. If the [=origin-sereialization=] of the target user agent's [=origin=] is not |serialized|.\[[Origin]], then throw a "<a exception>DataCloneError</a>" {{DOMException}}.
```

Comment:
Does `throw a DataCloneError` here mean that if you postmessage a wasm module, and a `DataCloneError` is thrown at the receiver, then the receiver gets a `messageerror` event instead of an `onmessage` event?